drivers/st7735/Kconfig

```
@@ -26,6 +26,12 @@ config MODULE_ST7789
     bool "ST7789 display driver"
     select MODULE_ST7735
 
+config ST77XX
+    bool "Select ST7789 if HAVE_ST7789, select ST7735 otherwise"
+    select MODULE_ST7789 if HAVE_ST7789
+    select MODULE_ST7735 if !HAVE_ST7789
+
```

I guess this should be an unvisible option
```suggestion
    bool
    select MODULE_ST7789 if HAVE_ST7789
    select MODULE_ST7735 if !HAVE_ST7789
    help
         Select ST7789 if HAVE_ST7789, select ST7735 otherwise

```